server/setup.js

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs';
+
+const { mkdir, writeFile } = fs.promises;
+
+export async function setup(path) {
+  await mkdir(`${path}/database`);
```

Comment:
let's rather use [ensureDir](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/ensureDir.md) instead of `mkdir`